config/site.yaml

```diff
@@ -0,0 +1,4 @@
+---
+region_name: CHI_DEV_UC
+image_container: pdmars-image-test-1
+scope: prod
```

Comment:
which config items must be specified here, and which via cmdline argument?

Reply:
For this tool I just put them all in a site.yaml since that is what the other implementation pulled from for what it needed (generated via the jinja template afair). In the build/push tool, they are just args to the cli, we can definitely do the same here.

If we go with a site.yaml I should probably move the object store cloud and image store cloud to this file to reduce the additional cli args when we already have this. Those got a bit one-off'd when I added them once I realized we needed to support different clouds for each.

Reply:
I moved more of them into the site.yaml